packages/monorepo-scripts/src/utils/npm_utils.ts

```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch';
+import * as _ from 'lodash';
+
+import { PackageRegistryJson } from '../types';
+
+const NPM_REGISTRY_BASE_URL = 'https://registry.npmjs.org';
+const SUCCESS_STATUS = 200;
```

Comment:
https://www.npmjs.com/package/http-status-codes
Let's stop hard-coding it in every place

Reply:
Will be better for grepability

Reply:
To do this refactor requires editing many files, across packages. I'd rather we do that on `v2-prototype`. I agree with you but will be leaving this for now.